salt/profiles/config/etc-nginx-sites-enabled-orcid-dummy.conf

```
@@ -0,0 +1,33 @@
+upstream orcid {
+    # elife-sidecars/Docker maps host port 8001 to guest port 8080
+    server localhost:8001;
+}
+
+server {
+    listen 8081;
+    listen 8082 ssl;
+
+    server_name localhost;
+
+    location = /favicon.ico { access_log off; log_not_found off; }
+    location = /robots.txt { access_log off; log_not_found off; }
+
+    location / {
```

```suggestion
    location / {
        proxy_pass http://orcid;
    }
```
and delete all the other `location` I think. This was all for the FastCGI protocol but with HTTP it should just need to forward.

---

(what new trickery is this? pressing the 'commit suggestion' button just cos)

---

yeah ... I think github was being too clever here.

---

(I wasn't able to do a multi-line suggestion to replace more than this line, but what it does is it would create a new commit with the same metadata used on merging PRs. Message fully customizable)